module/modmanager/manager_test.go

```diff
@@ -32,6 +33,20 @@ import (
 	rutils "go.viam.com/rdk/utils"
 )
 
+type TestDiscoveryResult struct {
+	Discovery []TestDiscoveryItem `json:"discovery"`
+}
+
+type TestDiscoveryItem struct {
+	Query   TestDiscoveryQuery `json:"query"`
+	Results map[string]string  `json:"results"`
+}
+
+type TestDiscoveryQuery struct {
+	Subtype string `json:"subtype"`
+	Model   string `json:"model"`
+}
```

Comment:
i think we can keep these private

```suggestion
type testDiscoveryResult struct {
	Discovery []testDiscoveryItem `json:"discovery"`
}

type testDiscoveryItem struct {
	Query   testDiscoveryQuery `json:"query"`
	Results map[string]string     `json:"results"`
}

type testDiscoveryQuery struct {
	Subtype string `json:"subtype"`
	Model   string  `json:"model"`
}
```